crates/libs/bindgen/src/implements.rs

```diff
@@ -1,86 +1,96 @@
 use super::*;
 
-pub fn gen(def: &TypeDef, gen: &Gen) -> TokenStream {
-    if def.kind() != TypeKind::Interface || (!gen.component && !def.can_implement()) {
+pub fn gen(gen: &Gen, def: TypeDef) -> TokenStream {
+    if gen.reader.type_def_kind(def) != TypeKind::Interface || (!gen.component && !gen.reader.type_def_can_implement(def)) {
         return quote! {};
     }
 
-    if def.name().starts_with("Disp") && def.methods().next().is_none() {
+    if gen.reader.type_def_name(def).starts_with("Disp") && gen.reader.type_def_methods(def).next().is_none() {
         return quote! {};
     }
 
-    let type_ident = gen_ident(def.name());
+    let type_name = gen.reader.type_def_type_name(def);
+    // TODO: workaround for https://github.com/microsoft/win32metadata/issues/814
+    if type_name.name == "INetCfgComponentUpperEdge" {
+        return quote! {};
+    }
```

Comment:
I believe this can be removed now. [Was fixed in a release in March](https://github.com/microsoft/win32metadata/commit/1939efb65d8c19f5301f5bb47fcd4087d51d47b9).

Reply:
They claim to have removed it but it's back. Anyway, I excluded it here to make diffing simpler, but I can now remove this exclusion.